test/crd.go

```diff
@@ -219,33 +218,18 @@ func LatestServiceLegacy(names ResourceNames, options *Options, fopt ...v1alpha1
 // called for the first time.
 var AppendRandomString = helpers.AppendRandomString
 
+// MakeK8sNamePrefix will convert each chunk of non-alphanumeric character into a single dash
+// and also convert camelcase tokens into dash-delimited lowercase tokens.
+var MakeK8sNamePrefix = helpers.MakeK8sNamePrefix
```

Comment:
any reason we create this additional var?

Reply:
I presume to follow suite above. But probably none, except saving import statement.

Reply:
Yeah just make it look similar with `AppendRandomString`, as both of them are imported from `pkg/test/helpers`.

Reply:
Any reason not to do [this](https://golang.org/ref/spec#Import_declarations)?
```go
import . "github.com/knative/pkg/test/helpers"
```